dropwizard-metrics/3.1/pom.xml

```diff
@@ -0,0 +1,35 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.wavefront</groupId>
+    <artifactId>wavefront</artifactId>
+    <version>3.7-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+  
+  <artifactId>metrics-wavefront</artifactId>
```

Comment:
dropwizard-metrics instead of metrics-wavefront? (it's odd when the package is com.wavefront.metrics-wavefront with two wavefronts)